pkg/networkservice/common/interpose/server.go

```diff
@@ -144,23 +144,8 @@ func (l *interposeServer) getConnectionID(conn *networkservice.Connection) (stri
 }
 
 func (l *interposeServer) Close(ctx context.Context, conn *networkservice.Connection) (*empty.Empty, error) {
-	logger := log.FromContext(ctx).WithField("interposeServer", "Close")
-
-	// If we came from NSMgr, we need to go to proper interpose NSE
-	connInfo, ok := l.activeConnection.Load(conn.GetId())
-	if !ok {
-		logger.Errorf("no active connection found: %v", conn)
-		return next.Server(ctx).Close(clienturlctx.WithClientURL(ctx, nil), conn)
-	}
-
-	var crossCTX context.Context
-	if conn.GetId() == connInfo.clientConnID {
-		crossCTX = clienturlctx.WithClientURL(ctx, connInfo.interposeNSEURL)
```

Comment:
Why we don't need more to pass clienturl to interposeNSE?

Reply:
Removed commit modifying interpose, guess it is no more part of the PR.